Sources/Testing/Running/Runner.Plan.swift

```diff
@@ -347,6 +347,29 @@ extension Runner.Plan {
   }
 }
 
+extension Runner.Plan.Action.RunOptions: Codable {
+  private enum CodingKeys: CodingKey {
+    case isParallelizationEnabled
+  }
+
+  public init(from decoder: any Decoder) throws {
```

Comment:
I _think_ the compiler can synthesize the `init` and `encode` members so long as the keys are defined. Compare the keys in `__CommandLineArguments_v0`. (I think you need to explicitly state a raw type of `String` too.)

Reply:
> 🛑 CodingKey case 'isParallelizationEnabled' does not match any stored properties

Dang it. Never mind.